### DISCUSSION TITLE
How to create OPTION message?

### DISCUSSION THREAD

**Discussion by zcc0077** [emiago/sipgo#80] [Q&A] [answered]
Hi, I used exosip2 to create options by `eXosip_options_build_request` to do a simple healthcheck, I plan to migrate to go and plan to use sipgo to do this, but I cannot figure out how to send OPTION msg to server, can you give me some hints? Big thanks :)

**Comment by emiago** [comment, marked as answer]
simple. 

Create request  with OPTION method and recipient
`req := sip.NewRequest(....)`

create client (you have that on readme), and just do transaction request 
`tx, err := client.TransactionRequest(ctx, req)` 

on tx you have Respond channel to grab response, rest you can ignore in this case. 

Pls close if you figure this out,

**Comment by zcc0077** [comment]
Thanks for the quick reply, I'll try ~~~~

**Comment by zcc0077** [comment]
solved ~~
